lib/locales/igbo/index.js

```diff
@@ -6,3 +6,4 @@ ig.title = 'Igbo';
 ig.name = require('./names');
 ig.lorem = require('./lorem');
 ig.address = require('../common/address');
+ig.internet = require('../common/internet');
```

Comment:
Is this class going to be different per locales? @rotimi-babalola

Reply:
I don't think so. I was uncomfortable with doing this too. Will fix

Reply:
Although we did something similar for `address` @BolajiOlajide 🤔

Reply:
I'll just import the `internet` folder directly in the internet class 😎